### PR TITLE
[1847 AE] Move to production

### PR DESCRIPTION
**Pull request by Galatolol**
## Before clicking "Create"

- [x] Branch is derived from the latest `master`
- [x] Code passes linter with `docker compose exec rack rubocop -a`
- [x] Tests pass cleanly with `docker compose exec rack rake`

### Explanation of Change

Move to production since there weren't any new issues for a few months.

### CODE REVIEW

File: assets/app/view/welcome.rb [hunk 1]
```diff
@@ -18,6 +18,7 @@ def render_notification
       message = <<~MESSAGE
         <p><a href='https://www.kickstarter.com/projects/1840vienna/18esp/'>18ESP is now on Kickstarter</a>.</p>
         <p><a href="https://github.com/tobymao/18xx/wiki/18Neb">18Neb</a> is in alpha.</p>
+        <p><a href="https://github.com/tobymao/18xx/wiki/18NL">1847 AE</a> is in production.</p>
```
Comment (ollybh) [inline assets/app/view/welcome.rb:21]: Wrong URL for wiki page.
```suggestion
        <p><a href="https://github.com/tobymao/18xx/wiki/1847-AE">1847 AE</a> is in production.</p>
```

**Review by ollybh — CHANGES_REQUESTED** [review]

File: assets/app/view/welcome.rb (hunk 1 above)
Comment (Galatolol) [reply assets/app/view/welcome.rb]: Thanks, fixed.

**Review by ollybh — COMMENTED** [review]
@Galatolol There is one open issue for 1847: #10473. It would be best to fix this (or close it if it's invalid) before moving this game to production.

**Comment by Galatolol** [comment]
Ah, I missed that issue. So I'll close the PR for now.

**Pull request closed without merging**